### PR TITLE
Integrate search into CI

### PR DESCRIPTION
**Pull request by varunarora**

### CODE REVIEW

File: portal/portal/management/commands/rebuild_index.py [hunk 1]
```diff
@@ -112,7 +112,7 @@ def build_document(self, source_dir, lang):
                         if title:
                             raw_title = next(title.stripped_strings)
 
-                            if lang == 'zh'
+                            if lang == 'zh':
```
Comment (jetfuel) [inline portal/portal/management/commands/rebuild_index.py]: lol, how did this work before

File: portal/portal/templates/_base.html [hunk 2]
```diff
@@ -62,8 +62,8 @@
         <script src="//twitter.github.com/typeahead.js/releases/latest/typeahead.jquery.min.js"></script>
 
         <script src="/static/js/lunr.js"></script>
-        <script src="/static/indexes/{{ lang }}/{{ CURRENT_DOCS_VERSION }}/index.js"></script>
-        <script src="/static/indexes/{{ lang }}/{{ CURRENT_DOCS_VERSION }}/toc.js"></script>
+        <script src="/static/{{ lang }}/{{ CURRENT_DOCS_VERSION }}/index.js"></script>
+        <script src="/static/{{ lang }}/{{ CURRENT_DOCS_VERSION }}/toc.js"></script>
```
Comment (jetfuel) [inline portal/portal/templates/_base.html]: why not keep the indexes path?

File: scripts/deploy/deploy_docs.sh [hunk 3]
```diff
@@ -28,7 +28,27 @@ export ENV=production
...
+
+# Merge the documentations, and temporarily swap spots.
+mkdir indexable-documentation
+rsync -r content_mgr@$STAGE_DEPLOY_IP:/var/pages/documentation indexable-documentation/
+cp -rf documentation/ indexable-documentation
+mv documentation new-documentation
+mv indexable-documentation documentation
+
+echo "5. Build the search index of the newly generated documentation."
+python manage.py rebuild_index en $GITHUB_BRANCH
```
Comment (jetfuel) [inline scripts/deploy/deploy_docs.sh]: see if we can capture the version from branch 'release-<version>'

File: portal/portal/templates/_base.html (hunk 2 above)
Comment (varunarora) [reply portal/portal/templates/_base.html]: That would create the dir structure `indexes/indexes/en...`

Would that work? Or do you prefer this?

**Comment by varunarora** [comment]
@jetfuel Ready for review and testing with `develop_doc` with your help. The indexes for all versions have been placed on the server.

File: portal/portal/management/commands/utils.py [hunk 4]
```diff
@@ -0,0 +1,7 @@
+def sanitize_version(raw_version):
```
Comment (jetfuel) [inline portal/portal/management/commands/utils.py]: This is great!

**Pull request merged**